src/test/scala/org/broadinstitute/dsde/workbench/sam/azure/MockCrlService.scala

```diff
@@ -0,0 +1,115 @@
+package org.broadinstitute.dsde.workbench.sam.azure
+
+import cats.effect.IO
+import com.azure.core.management.Region
+import com.azure.core.util.Context
+import com.azure.resourcemanager.msi.MsiManager
+import com.azure.resourcemanager.msi.models.{Identities, Identity}
+import com.azure.resourcemanager.msi.models.Identity.DefinitionStages
+import com.azure.resourcemanager.resources.ResourceManager
+import com.azure.resourcemanager.resources.fluent.models.ResourceGroupInner
+import com.azure.resourcemanager.resources.models.{GenericResource, GenericResources, Plan, ResourceGroup, ResourceGroups}
+import org.broadinstitute.dsde.workbench.sam.model.{FullyQualifiedResourceId, ResourceId, SamResourceTypes}
+import org.mockito.ArgumentMatchers
+import org.mockito.ArgumentMatchers.{any, anyString}
+import org.mockito.Mockito.when
+import org.scalatestplus.mockito.MockitoSugar
+
+import scala.jdk.CollectionConverters._
+
+object MockCrlService extends MockitoSugar {
```

Comment:
Hopefully this mockito usage isn't too horrible. 😂  Azure SDKs are mock-able, but create a lot of intermediate classes which need to be mocked.